packages/lib/invite/service.ts

```diff
@@ -62,27 +73,32 @@ export const deleteInvite = async (inviteId: string): Promise<TInvite> => {
   return deletedInvite;
 };
 
-export const getInviteToken = cache(async (inviteId: string) => {
-  const invite = await prisma.invite.findUnique({
-    where: {
-      id: inviteId,
-    },
-    select: {
-      email: true,
-    },
-  });
+export const getInviteToken = cache(
```

Comment:
`getInviteToken` is essentially a `getInvite`. Can you please remove `getInviteToken` and make sure to return only the required data in the action where it's used. that way it's also much easier for us to use caching.

Reply:
Done